server/src/main/java/org/apache/druid/segment/realtime/FireDepartmentMetrics.java

```diff
@@ -113,6 +114,9 @@ public void incrementMergeTimeMillis(long millis)
   {
     mergeTimeMillis.addAndGet(millis);
   }
+  public void incrementMergeRows(long rows) {
```

Comment:
Can you please document what merge rows mean's ?

Reply:
Added a comment to the variable